### PR TITLE
EdFi v0.2

### PR DESCRIPTION
**Pull request by dblickwedel**

### CODE REVIEW

**Review by Jessuss21 — APPROVED** [review]

File: framework/synapse/notebook/OEATestKit_py.ipynb [hunk 1]
```diff
@@ -1,26 +1,12 @@
...
-      "version": "0.1",
-      "state": {}
-    }
-  },
   "cells": [
     {
       "cell_type": "code",
-      "execution_count": null,
+      "source": [
+        "%run /OEA_py"
```
Comment (Jessuss21) [inline framework/synapse/notebook/OEATestKit_py.ipynb:6]: cf49ef37cc3ee243a8e3b4621dfd5728b78dfc85

File: modules/module_catalog/Ed-Fi/setup.sh [hunk 2]
```diff
@@ -2,41 +2,28 @@
 
 # Installs the Ed-Fi module
 # This script can be invoked directly to install the Ed-Fi module assets into an existing Synapse Workspace.
-if [ $# -ne 2 ]; then
-    echo "This setup script will install the Ed-Fi module assets into an existing Synapse workspace."
+if [ $# -ne 1 ]; then
+    echo "This setup script will install the Microsoft Education Insights module assets into an existing Synapse workspace."
```
Comment (dastewa) [inline modules/module_catalog/Ed-Fi/setup.sh]: Ed-Fi vs Microsoft Education Insights?

File: framework/synapse/notebook/OEATestKit_py.ipynb (hunk 1 above)
Comment (Jessuss21) [reply framework/synapse/notebook/OEATestKit_py.ipynb:6]: 1. framework/synapse/notebook/OEATestKit_py.ipynb

**Review by dastewa — APPROVED** [review]

**Review by Jessuss21 — APPROVED** [review]
Quiero un desarrollo personal ami venegicio

File: framework/synapse/notebook/OEATestKit_py.ipynb (hunk 1 above)
Comment (dblickwedel) [reply framework/synapse/notebook/OEATestKit_py.ipynb:6]: I'm not sure what the comment is here

**Pull request merged**